### ISSUE TITLE
Simple relation

### ISSUE THREAD

**Issue by MichiganRx** [dimatkach11/strapi_plugins#6]
I have 3 collections: states, cities, and offers. I register the states, and the registered cities are related to the states. In the offer, I have a select that lists all the states and filters the city select (it will list all the cities of that state), but I need to select multiple cities.
I couldn't understand the other example.

**Comment by MichiganRx** [comment]
Can you please help me? I've been trying to do this for two days.

**Comment by dimatkach11** [comment]
Let's assume I have some records inserted into the database with the following structure (since I don't have other details):

### Collections Schema

#### Nations ( By mistake I called the collection Nations instead of States )

| name          | locale_code | other_field |
| ------------- | --------- | ----------- |
| Italy         | it_It     | ...         |
| United States | en_US     | ...         |
| Germany       | de_DE     | ...         |

#### Cities

| name        | city_code | other_field |
| ----------- | ----------- | ----------- |
| Rome        | it_It       | ...         |
| Milan       | it_It       | ...         |
| Naples      | it_It       | ...         |
| New York    | en_US       | ...         |
| Los Angeles | en_US       | ...         |
| California  | en_US       | ...         |
| Chicago     | en_US       | ...         |
| Berlin      | de_DE       | ...         |
| Frankfurt   | de_DE       | ...         |

### Offers Content Type Builder Configuration

- fields overview
![issue_6-1](https://github.com/user-attachments/assets/b4d56b70-cfa2-4826-a71b-1d8a3ea3784f)


#### Custom Field: CustomRelation - Configuration

- `c_nation` basic setting:
![issue_6-2](https://github.com/user-attachments/assets/2ba2db43-fe58-4608-8ed2-0540060710df)

- `c_nation` advanced setting:
![issue_6-3](https://github.com/user-attachments/assets/b943ef8e-4acd-4b35-9b66-905410241e94)

- `c_cities` basic setting:
![issue_6-4](https://github.com/user-attachments/assets/c5e43696-6565-40f5-97f0-30ebadb612a6)

- `c_cities` advanced setting:
![issue_6-5](https://github.com/user-attachments/assets/02b013e9-6035-4051-b148-f75b66792d7c)

### Offers Content Manager 

![issue-6](https://github.com/user-attachments/assets/87be5799-60bd-4610-8219-b0e23cd206c3)

**Let me know if it's clearer now or if you need more help!**

**Comment by MichiganRx** [comment]
Thank you so much for the help! Just one more question: can't I make a component with it? Everything worked as you explained, but then I tried to make a repeatable component, and it seems like it doesn't accept it.

![image](https://github.com/user-attachments/assets/26bab023-62d7-4e05-ba54-38d058db994f)

**Comment by MichiganRx** [comment]
When I use it directly in the collection, it works correctly.

![image](https://github.com/user-attachments/assets/2bb70d1f-319b-4402-9a15-952e9f03c282)

**Comment by MichiganRx** [comment]
Sorry if I'm writing it wrong, I only know how to speak Portuguese and I'm using a translator.

**Comment by dimatkach11** [comment]
> Thank you so much for the help! Just one more question: can't I make a component with it? Everything worked as you explained, but then I tried to make a repeatable component, and it seems like it doesn't accept it.
> 
> ![image](https://private-user-images.githubusercontent.com/111995632/374690073-26bab023-62d7-4e05-ba54-38d058db994f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0NTE3MDcsIm5iZiI6MTcyODQ1MTQwNywicGF0aCI6Ii8xMTE5OTU2MzIvMzc0NjkwMDczLTI2YmFiMDIzLTYyZDctNGUwNS1iYTU0LTM4ZDA1OGRiOTk0Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwOVQwNTIzMjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kOGEzOGZiNTk1ZDI0NGU2ODE4MzQ2YjI1ZWI3OWI1NjVjMzFjNmM5ZDhiZmMxZWMxNGUzOTNlNWQzNGQzNmJjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.FZHC5wZJ2GUT3bciQZ9ypMz-rANocJVX7hYzpKF9mWk)

Yes, as you rightly pointed out, unfortunately the functionality is still missing in the components. As soon as I introduce it, I will update this issue. Thanks for reporting this lack
